.circleci/config.yml

```diff
@@ -1811,20 +1835,51 @@ jobs:
       workflow-name:  << parameters.workflow-name >>
     steps:
       - run:
-          name: Modify Payer account for Migration test
+          name: Modify Payer account for Public testnet Migration test
           command: |
             sed -i 's/default.payer=0.0.2/default.payer=0.0.50/g' /repo/test-clients/src/main/resource/spec-default.properties;
             echo -n "$KEY_50" > /repo/test-clients/src/main/resource/StartUpAccount.txt;
 
       - run:
-          name: Run Migration test
+          name: Run Public testnet Migration test
           command: |
             cd /repo ; mvn --no-transfer-progress clean install -DskipTests;
             cd /swirlds-platform/regression;
             ./regression_services_circleci.sh AWS-Services-Daily-Migration-4N-1C.json /repo
 
       - run:
-          name: Sync results of migration test to AWS
+          name: Sync results of public testnet migration test to AWS
+          command: |
+            aws s3 sync /swirlds-platform/regression/results/ s3://hedera-service-regression-jrs;
+            tar -czvf /results.tar.gz  /swirlds-platform/regression/results/*
+
+      - store_artifacts:
+          path: /results.tar.gz
+
+  run-mainnet-migration-regression:
+    parameters:
+      workflow-name:
+        type: string
+        default: ""
+    executor:
+      name: build-executor
+      workflow-name:  << parameters.workflow-name >>
+    steps:
+      - run:
+          name: Modify Payer account for Mainnet Migration test
+          command: |
+            sed -i 's/default.payer=0.0.2/default.payer=0.0.950/g' /repo/test-clients/src/main/resource/spec-default.properties;
+            echo -n "$KEY_950" > /repo/test-clients/src/main/resource/StartUpAccount.txt;
```

Comment:
Just a word of caution : KEY_950 should be  added to circleCI

Reply:
Yeah, already added and tested with the workflow.